utils.py

```diff
@@ -283,3 +283,6 @@ def isoformat(timestamp):
     """Turn a timestamp into an ISO formatted string."""
     dt = datetime.datetime.utcfromtimestamp(timestamp)
     return dt.isoformat() + 'Z'
+
+def is_heroku():
+    return __name__ == '__main__'
```

Comment:
This won't work. `__name__` has a meaning local to a file, and it means "the module name by which the *current* file was imported." For this file, it will always be `"utils"`.

That's why it works as a trick in the entrypoint file: since the entrypoint file is NOT actually imported, it has a magic special value there (`"__main__"`).

See:

* https://docs.python.org/3/library/__main__.html
* https://stackoverflow.com/questions/419163/what-does-if-name-main-do

I have an idea though. Give me a moment.

Reply:
Oh, great catch. That's on me for not digging a bit deeper on what that means.